Modules/Core/Common/include/itkArray2D.h

```diff
@@ -79,14 +79,15 @@ class ITK_TEMPLATE_EXPORT Array2D : public vnl_matrix<TValue>
   /** Move-assignment operator.
    * \note This move-assignment operator is `noexcept`, even while the move-assignment operator of its base class
    * (`vnl_matrix`) is not `noexcept`, because unlike `vnl_matrix`, `Array2D` always manages its own memory. */
+  /** @ITKStartGrouping */
   Self &
   operator=(Self && array) noexcept
   {
     // Note: GCC <= 9.5 does not yet support "defaulting" (`= default`) this `noexcept` move-assignment operator.
     this->VnlMatrixType::operator=(std::move(array));
     return *this;
   }
-
+  /**@ITKEndGrouping*/
```

Comment:
Is there some way to use doxygen to detect these single-member groups?

Reply:
No there isn't, maybe it is possible to tweak a local copy of doxygen so it spits out a debug line for these cases (though I'm not to optimistic about it).